### PR TITLE
Ruby 2.6.0-preview2 Released

### PR DESCRIPTION
**Pull request by nurse**

### CODE REVIEW

File: en/news/_posts/2018-05-31-ruby-2-6-0-preview2-released.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  x1.4 improvements [Bug #10212].
+
+* Speedup `block.call` where `block` is passed block parameter. [Feature #14330]
+  Ruby 2.5 improves block passing performance. [Feature #14045]
+  Additionally, Ruby 2.6 improves the performance of passed block calling.
+  With micro-benchmark we can observe x2.6 improvemnt.
+
+## Other notable changes since 2.5
+
+* Range#cover? いる？
```
Comment (yous) [inline en/news/_posts/2018-05-31-ruby-2-6-0-preview2-released.md]: The sentence is accidentally added.

File: en/news/_posts/2018-05-31-ruby-2-6-0-preview2-released.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+* Speedup `Proc#call` because we dont' need to care about `$SAFE` any more.
+  [Feature #14318]
+  With `lc_fizzbuzz` benchmark which uses `Proc#call` so many times we can measure
+  x1.4 improvements [Bug #10212].
+
+* Speedup `block.call` where `block` is passed block parameter. [Feature #14330]
+  Ruby 2.5 improves block passing performance. [Feature #14045]
+  Additionally, Ruby 2.6 improves the performance of passed block calling.
+  With micro-benchmark we can observe x2.6 improvemnt.
```
Comment (koic) [inline en/news/_posts/2018-05-31-ruby-2-6-0-preview2-released.md]: `improvemnt` seems to be a misprint of `improvement`.

File: en/news/_posts/2018-05-31-ruby-2-6-0-preview2-released.md (hunk 1 above)
Comment (nurse) [reply en/news/_posts/2018-05-31-ruby-2-6-0-preview2-released.md]: thanks! I'll remove it

File: ja/news/_posts/2018-05-31-ruby-2-6-0-preview2-released.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    ary.zip(1..) {|elem, index| ... } # ary.each.with_index(1) { ... }
+
+* Binding#source_location の追加 [Feature #14230]
+  * `binding`のソースコード上の位置を `__FILE__` と `__LINE__` の二要素配列として返します。従来でも `eval("[__FILE__, __LINE__]", binding)` とすることでこれらの情報は得られましたが、将来的に `Kernel#eval` はbindingのソースコード行を無視する変更を予定しているため [Bug #4352]、この新しいメソッドを用いることが今後は推奨されます。
+* Kernel#system の失敗時に、falseを返す代わりに例外を上げさせる :exception オプションを追加 [Feature #14386]
+
+## パフォーマンスの改善
+
+* 後述の$SAFEの変更に伴って考慮すべきことが減ったため、`Proc#call`が高速化されました [Feature #14318]
+  `Proc#call` を大量に呼び出す `lc_fizzbuzz` ベンチマークにおいては、1.4倍高速化されています [Bug #10212].
```
Comment (znz) [inline ja/news/_posts/2018-05-31-ruby-2-6-0-preview2-released.md]: `.` → `。` ?

File: en/news/_posts/2018-05-31-ruby-2-6-0-preview2-released.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+We're going to implement method iniling in JIT compiler, which is expected to increase Ruby's performance in order of magnitude.
+Also, we're planning to increase the supported platforms, and the next plan is to support Visual Studio.
+
+Stay tuned for the new age of Ruby's performance.
+
+## RubyVM::AST [Experimental]
+
+Ruby 2.6 introduces `RubyVM::AST` module.
+
+This module has `parse` method which parses a given string and returns AST (Abstract Syntax Tree) nodes, and `parse_file` method which parses a given file and returns AST nodes.
```
Comment (yui-knk) [inline en/news/_posts/2018-05-31-ruby-2-6-0-preview2-released.md]: Please update this to

```
This module has `parse` method which parses a given ruby code of string and returns AST (Abstract Syntax Tree) nodes, and `parse_file` method which parses a given ruby code file and returns AST nodes.
```

File: en/news/_posts/2018-05-31-ruby-2-6-0-preview2-released.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+`RubyVM::AST::Node` class is also introduced you can get location information and children nodes from `Node` objects. This feature is experimental, so compatibility is not guaranteed.
```
Comment (yui-knk) [inline en/news/_posts/2018-05-31-ruby-2-6-0-preview2-released.md]: Please update this to

```
`RubyVM::AST::Node` class is also introduced you can get location information and children nodes from `Node` objects. This feature is experimental. Compatibility of the structure of AST nodes are not guaranteed.
```

**Pull request merged**